### ISSUE TITLE
Replace SWF file with Movie I converted

### ISSUE THREAD

**Issue by mbanados** [tojrobinson/jurassicsystems.com#29]
Is there a reason theking.swf is flash instead of an embedded movie like .mp4? Not everybody has flash and Adobe is trying to abandon it. 

I converted the SWF file  and made 3 movies of different aspect ratios  in my fork https://github.com/mbanados/jurassicsystems.com/tree/master/static

I tired to incorporate them but sadly the audio would launch immidatly on the IRIX page instead of waiting for the Macintosh enviroment to load because I am not too familiar with the code. Perhaps you can make this happen? I can make the file any format needed.

**Comment by tojrobinson** [comment]
Yah this is definitely a better idea. I think if we just embed an mp4 with both the audio and video, that would be much better than a swf. Trying to sync audio and video of two separate files will be a nightmare though ofc. Will look into it. Thanks.

**Comment by mbanados** [comment]
Take a look at the three "theking*.mp4 " movies in the root of my fork mentioned above.. I guess I did not explain too well. the mp4 files have the audio built in to each file. I just had issues making the page not load the movie immediately. 

https://github.com/mbanados/jurassicsystems.com/tree/master/static

**Comment by mbanados** [comment]
the MP4 provided on my fork has the audio and video in it.

**Comment by PizzaHolic01** [comment]
We should probably get that .mp4 uploaded ASAP, cause Chrome is gonna terminate support for Flash on January 2020.

**Comment by mbanados** [comment]
https://github.com/mbanados/jurassicsystems.com/blob/master/static/theking720-4x3.mp4

**Comment by tojrobinson** [comment]
Hmm so autoplaying video with audio is kind of an issue here. Doesn't look like there's a nice cross browser solution that I can see. I guess it's better than the no load + allow Flash button experience at the moment.

**Comment by mbanados** [comment]
Confused.. the audio isnin the video... Referenced here .

**Comment by tojrobinson** [comment]
I mean, making an mp4 auto play with it's sound on is blocked by chrome and ff too I believe.

**Comment by PizzaHolic01** [comment]
Is there another option besides .mp4? Can you put an .avi and autoplay in JS?

**Comment by mbanados** [comment]
I have such a video on GitHub forked... Here is the link...  It is based
off the original flash..
https://github.com/mbanados/jurassicsystems.com/blob/master/static/theking720-4x3.mp4

**Comment by mbanados** [comment]
Infact it is small enough to enclose here ...

**Comment by SamueleCaprioli97** [comment]
I don't know if is helpful, but in my repo I have already done that. See my [commits](https://github.com/SamueleCaprioli97/jurassicsystems.com/commits/master).

**Comment by tojrobinson** [comment]
Pushed a commit just then and put it on the live site. The video looks a little low res but it's autoplaying fine on chrome and ff (you might need to hard refresh to get the new js & css files).

**Comment by mbanados** [comment]
I have a higher resolution version here based on the original flash..


https://github.com/mbanados/jurassicsystems.com/blob/master/static/theking720-4x3.mp4

**Comment by mbanados** [comment]
Do you know how to create a pull request with this higer resolution video..
I tried to contact Tully but no response

**Comment by tojrobinson** [comment]
@mbanados I'll look into updating to a higher res .mp4 this weekend
